src/script/src/python/pyo3/builtins.rs

```diff
@@ -90,7 +90,7 @@ pub(crate) fn greptime_builtins(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
             floor,
             ceil,
             round,
-            trunc,
+            // trunc,
```

Comment:
Do we need to add a TODO?

Reply:
Filed a issue https://github.com/GreptimeTeam/greptimedb/issues/2663